content/en/dora_metrics/deployments/_index.md

```diff
@@ -136,7 +136,10 @@ DORA Metrics for the service `shopist` only consider the Git commits that includ
 
 - Change lead time stage breakdown metrics are only available for GitHub.
 - Change lead time is not available for the first deployment of a service that includes Git information.
-- If commits on a feature branch are squashed into a single commit prior to being merged into the default branch, the commit history for that pull request is not included in the change lead time calculation. The first commit included in the calculation is the newly created single commit.
+- For rebased feature branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
```

Comment:
```suggestion
- For rebased branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
```
It would also be true if you do a rebase on the main/deployment branches (nobody would do this, but still)